dts/arm/nxp/nxp_rt6xx_ns.dtsi

```diff
@@ -6,8 +6,8 @@
 
 / {
 	soc {
-		sram: sram@20000000 {
-			ranges = <0x20000000 0x20000000 0x480000>;
+		sram: sram@20180000 {
+			ranges = <0x20180000 0x30180000 0x300000>;
```

Comment:
Second argument for the ranges should be 0x20180000

Reply:
@mmahadevan108  do you have the ranges setting, I think it is physical base, mapped address, mapped size, so this is to mapped the 0x20180000 to 0x30180000 length 0x30000, right?